docs/docs/components/selectors.mdx

```diff
@@ -110,10 +133,15 @@ Selectors predict a bot response from a set of candidate responses.
     set `weight_sparsity` to 1 as this would result in all kernel weights being 0, i.e. the model is not able
     to learn.
 
-  In addition, the component can also be configured to train a response selector for a particular retrieval intent.
-  The parameter `retrieval_intent` sets the name of the intent for which this response selector model is trained.
+  The component can also be configured to train a response selector for a particular retrieval intent.
+  The parameter `retrieval_intent` sets the name of the retrieval intent for which this response selector model is trained.
   Default is `None`, i.e. the model is trained for all retrieval intents.
 
+  In addition, the retrieval model can also be configured to use the text of the response messages as the training label
```

Comment:
```suggestion
  The retrieval model can also be configured to use the text of the response messages as the training label
```

Reply:
Instead of what? maybe this section should go under where you talked about how it works by default

Reply:
Actually moved the default behaviour to this section as this section talks about the configuration of the model and hence more appropriate.